src/common/ownsql.cpp

```diff
@@ -49,6 +49,12 @@ SqlDatabase::SqlDatabase()
 {
 }
 
+SqlDatabase::~SqlDatabase()
+{
+    close();
```

Comment:
I have a vague memory of us removing close-on-destruction before? I didn't find it in the revision history though and might be wrong.

Reply:
We already close it on destruction in the ~SyncJounrnalDb  
so this close is in fact a no-op.  I just felt it made sense in SqlDatabase itself, but i can remove it.

Reply:
No, then I'm wrong about this